apps/analogquadclk/app.js

```diff
@@ -31,12 +32,11 @@ const hand_hour_bg = get_hand(HOUR_LEN, HOUR_W, 8, HOUR_BACK);
 const hand_minute = get_hand(MIN_LEN, 4, 3, MIN_BACK);
 const hand_minute_bg = get_hand(MIN_LEN, MIN_W, 6, MIN_BACK);
 
-
 // schedule a draw for the next minute
 function queueDraw() {
   if (drawTimeout) clearTimeout(drawTimeout);
   drawTimeout = setTimeout(function() {
-    drawTimeout = undefined;
+    //drawTimeout = undefined;
```

Comment:
what's the reason for this one? It feels like a good thing to do in general, as at this point the timeout isn't valid any more.

Reply:
Oh, sorry - just spotted your reasoning for this. Got to say I'm not convinced about it - I'm not really sure I see how it can help